README.md

```diff
@@ -186,6 +186,22 @@ You can also declare `LoggingLevelSwitch`-es in custom section and reference the
 
 Level updates to switches are also respected for a dynamic update.
 
+Since version 4.0.0, both declared switches (i.e. `Serilog:LevelSwitches` section) and minimum level override switches (i.e. `Serilog:MinimumLevel:Override` section) are exposed through a callback on the reader options so that a reference can be kept:
+
+```csharp
+var allSwitches = new Dictionary<string, LoggingLevelSwitch>();
+var options = new ConfigurationReaderOptions
+{
+    OnAddedLevelSwitch = (switchName, logLevelSwitch) => allSwitches[switchName] = logLevelSwitch
```

Comment:
Nitpick, I think the other examples tend to call `LoggingLevelSwitch` parameters `levelSwitch`.

Reply:
Done in b483da5de199d764f0dc36fa2418bd086dee0715 too.